src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotMatchKeywords.cs

```diff
@@ -1,69 +1,621 @@
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
         internal static DiagnosticDescriptor NamespaceRule = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
                                                                              s_localizableMessageNamespace,
-                                                                             DiagnosticCategory.Naming,
-                                                                             DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             Category,
+                                                                             Severity,
+                                                                             isEnabledByDefault: IsEnabledByDefault,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
-                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+                                                                             helpLinkUri: HelpLinkUri,
+                                                                             customTags: CustomTags);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(MemberParameterRule, MemberRule, TypeRule, NamespaceRule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            
+            analysisContext.RegisterSymbolAction(AnalyzeMemberRule,
+                SymbolKind.Event, SymbolKind.Method, SymbolKind.Property);
+
+            analysisContext.RegisterSymbolAction(AnalyzeMemberParameterRule,
+                SymbolKind.Method);
+        }
+
+        private void AnalyzeMemberRule(SymbolAnalysisContext context)
+        {
+            ISymbol symbol = context.Symbol;
+            if (symbol.GetResultantVisibility() != SymbolVisibility.Public)
+            {
+                return;
+            }
+
+            string matchingKeyword;
+            if (!IsKeyword(symbol.Name, out matchingKeyword))
+            {
+                return;
+            }
+
+            // IsAbstract returns true for both abstract class members and interface members.
+            if (symbol.IsVirtual || symbol.IsAbstract)
```

Comment:
No, this is ok. I checked that FxCop does not report either `override` or `sealed override`. I already had a unit test for `override`, but I've now added one for `sealed override.

The rationale seems to be that if you're introducing a new `virtual` method, we'll catch you. But if you're overriding and existing method, there's nothing you can do about it -- you're stuck with the method name from the base class, so there's no point in bothering you about it. (Although it's true that in VB you can implement an interface method with a method of a different name.)